### ISSUE TITLE
Adding folder to upload config

### ISSUE THREAD

**Issue by amityweb** [sameer1750/cloudinary-upload#1]
I know this is extremely old and may not ever get updates, but this was the only Gulp to Cloudinary upload package I could find that works!! Fantastic. But there is one issue, and that is on the read me it does not specify an upload folder. So after some fiddling about I got it working with this:

```
var config = {
    cloud_name: 'test',
    api_key: '12345',
    api_secret: 'Abc-Xyz',
    folder: 'folder-to-upload-to'
};
```

**Comment by sameer1750** [comment]
what is the issue then??

**Comment by amityweb** [comment]
Well, firstly it's in "Issues" as its the only place I could give some feedback on this. And you do not mention the folder config setting, so it took me some time to find out and so thought I would add here in case others wanted to know, or you could add to your instructions. 
Thanks!

**Comment by sameer1750** [comment]
Ok thanks.. will try to add it .

**Comment by amityweb** [comment]
Oh one thing... I don't know for sure but if the config settings are Cloudinary config settings, because that's where I found the folder parameter, then a link to their page details all config settings would be good. You don't need to detail the config and there's so much more. For example I invalidate the cache when uploading. Things like that.

**Comment by amityweb** [comment]
I got the parameters from here:
https://cloudinary.com/documentation/image_upload_api_reference#upload

Sorry if it's implied they're the same, I didn't think at the time. So a link to this is a big help. 

I'm surprised there's a lack of gulp upload to CDN packages. Yours is the only one that worked. They're all really old too. Hardly any, if any, people using them (I assume from lack of issues). Yet this is a brilliant package. 

My use case is I have a Vercel Next.js site which takes ages to build. If I change like one css style I have to wait ages for the deploy process just for that. So I decided to put stylesheets on a  CDN (customer uses Cloudinary) and link the site to those. Now when I change css I don't have to deploy the whole site, it just updates after a several sends or a bit more. Using Gulp means I don't need to manually upload. I change cas, and it gets compressed etc and uploaded to the CDN. Brilliant.
